src/Jackett.Common/Indexers/SceneTime.cs

```diff
@@ -172,43 +171,43 @@ private List<ReleaseInfo> ParseResponse(TorznabQuery query, string htmlResponse)
                 var rows = dom.QuerySelectorAll("tr.browse");
                 foreach (var row in rows)
                 {
-                    // TODO convert to initializer
-                    var release = new ReleaseInfo
-                    {
-                        MinimumRatio = 1,
-                        MinimumSeedTime = 259200 // 72 hours
-                    };
-
-                    var catId = "82"; // default
-                    var qCatLink = row.Children[categoryIndex].QuerySelector("a");
-                    if (qCatLink != null)
-                    {
-                        catId = new Regex(@"\?cat=(\d*)").Match(qCatLink.GetAttribute("href")).Groups[1].ToString().Trim();
-                    }
-                    release.Category = MapTrackerCatToNewznab(catId);
```

Comment:
This was added in https://github.com/Jackett/Jackett/commit/bef73a5f50dfef9739571bb1b1a2f07675d0caa5 to handle missing categories. It appears to be still be an issue. If you search `blue` you get the following error:

```text
2023-01-29 05:40:17 | Error | Error on 20230129054017 for SceneTime.txtSystem.NullReferenceException: Object reference not set to an instance of an object.    at Jackett.Common.Indexers.SceneTime.ParseResponse(TorznabQuery query, String htmlResponse) in D:\a\1\s\src\Jackett.Common\Indexers\SceneTime.cs:line 183
2023-01-29 05:40:17 | Error | Jackett.Common.IndexerException: Exception (scenetime): Object reference not set to an instance of an object.
```